lua/nvim-treesitter/shell_command_selectors.lua

```diff
@@ -172,4 +183,20 @@ function M.select_download_commands(repo, project_name, cache_folder, revision)
   end
 end
 
+function M.select_temp_directory_change_command_start(dir)
+  if fn.has('win32') == 1 then
+    return string.format("pushd %s & ", dir)
```

Comment:
Will this always work on win32? Even when using bash?

Reply:
hmm I guess not. this one is tricky

Reply:
Maybe Developer Command Prompt sets some env variables? Does something like `$SHELL` exists on Windows.

Reply:
hmm so `pushd` and `popd` do work locally on Mac, but not on CI in ubuntu.

Reply:
checked those commands in Git Bash and they should work. was able to compile `cpp` with clang.